canvas.py

```diff
@@ -0,0 +1,94 @@
+from random import random
+from kivy.app import App
+from kivy.uix.widget import Widget
+from kivy.uix.button import Button
+from kivy.graphics import Color, Ellipse, Line
+from kivy.uix.floatlayout import FloatLayout
+from functools import partial
+global X, Y
+#Initial coords.
+X = 400
+Y = 400
+
+class Canvas(Widget):
+
+    def paint(self, token, event):
+        color = (1, 1, 1)  #Red Color
+        global X, Y
+
+        #Increase/Decrease of coordinates dependant on token value passed on button press
+
+        if(token == 0):
```

Comment:
Consider putting these in a if-else block

Reply:
Sure, but since it's small scale processing, doesn't make much difference in overall performance.